src/marketing/url-rewrite-configuration.md

```diff
@@ -26,3 +26,21 @@ URL with Web Server Rewrite
 1. Set **Use Web Server Rewrites** to your preference.
 
 1. When complete, click <span class="btn">Save Config</span>.
+
+## Use canonical URLs
+
+It is imperative that each of your web pages only has one separate URL for SEO consolidation.
+
+If you have a single page accessible by multiple URLs, or different pages with similar content, Google sees these as duplicate versions of the same page. Google will choose one URL as the canonical version and crawl that, and all other URLs will be considered duplicate URLs and crawled less often.
+
+If you don't explicitly tell Google which URL is canonical, Google will make the choice for you, or might consider them both of equal weight, which might lead to unwanted behavior, run the risk of an ineffective crawl budget and low distributed backlinks.
```

Comment:
This is a long sentence. Break it up into two.

Reply:
Done ✅